src/components/MarkerButton/index.tsx

```diff
@@ -10,13 +9,12 @@ interface MarkerButtonProps {
 }
 
 const MarkerButton: FC<MarkerButtonProps> = ({ item }) => {
-  // @ts-ignore
-  const { sidebarIsOpen, setCurrentMonument, onOpen, monument } : SidebarContextInterface | {} = useContext(SidebarContext);
-  const isActive = sidebarIsOpen && monument.id === item.id;
+  const params: { id?: string } = useParams();
+  const isActive = params?.id === item.id;
+  let history = useHistory();
```

Comment:
я бы вот так сделал, хотя в доке почему-то тоже `let` используется

```suggestion
  const history = useHistory();
```